hooks/blueprint

```diff
@@ -92,6 +93,11 @@ for want in ${GENESIS_REQUESTED_FEATURES}; do
         manifests/addons/external-db-internal-db-cleanup.yml \
         manifests/addons/external-db.yml 
       )
+      if want_feature external-db-mysql; then
```

Comment:
its cleaner to have external-db-mysql to be stand-alone, instead of a sub-feature of external-db.

Have line 91 be:
```
if want_feature external-db || want_feature external-db-mysql; then
```

Reply:
and update the docs to indicate they don't have to specify both